src/main/java/org/sonarsource/sonarlint/ls/SonarLintLanguageServer.java

```diff
@@ -278,6 +278,7 @@ private static TextDocumentSyncOptions getTextDocumentSyncOptions() {
       serverNotifications.shutdown();
       standaloneEngineManager.shutdown();
       moduleEventsProcessor.shutdown();
+      branchManager.stop();
```

Comment:
I took the habit to name methods "shutdown" to make it clear they will be called by the "shutdown" of the LSP